### ISSUE TITLE
service weights needs integer

### ISSUE THREAD

**Issue by maxadamo** [solarkennedy/puppet-consul#492]
starting from Consul 1.6.x, the service weight requires an integer, but since this is configured with the extra hash `service_config_hash`, I am not being to set the integer in the json file. 

This is what I had in my puppet manifest (I have also tried the trick of using `0 +`, to force integer, but it didn't work:

```
  consul::service { 'puppet':
    service_name        => 'puppet',
    enable_tag_override => false,
    checks              => [
      {
        args     => ['/usr/local/bin/puppet-check.sh'],
        interval => '10s'
      }
    ],
    port                => 8140,
    tags                => [
      '_x-puppet._tcp',
      '_x-puppet-fileserver._tcp'
    ],
    service_config_hash =>  {
      weights => {
        passing => 10,
        warning => 1
      }
    }
  }
```

**Comment by bastelfreak** [comment]
Hi. We already have something to not quote certain elements:
https://github.com/solarkennedy/puppet-consul/blob/714756d1fa0e72e4759af5deca453d33935b8be2/lib/puppet/functions/consul/sorted_json.rb#L139-L141

Could you check if you can adopt it for weights as well?

**Comment by maxadamo** [comment]
lemme-check!

**Comment by maxadamo** [comment]
lemme-fork :-)

**Comment by maxadamo** [comment]
fixed by #493 (if Travis likes it)

**Comment by maxadamo** [comment]
@solarkennedy  fixed by #509
